data/schema.js

```diff
@@ -126,6 +126,17 @@ var schema = {
       created_date:            { type: 'dateTime', nullable: false },
       updated_date:            { type: 'dateTime', nullable: false }
     }
+  },
+
+  nfc_tag: {
+    primary: 'id',
+    columns: {
+      id:                      { type: 'uuid', nullable: false },
+      ribot_id:                { type: 'uuid', nullable: false, references: { table: 'ribot', column: 'id' } },
+      uid:                     { type: 'text', nullable: false },
```

Comment:
This is different in the migration. They should be identical to avoid bugs in the future.